src/OrchardCore/OrchardCore.Indexing.Abstractions/ContentPartIndexHandler.cs

```diff
@@ -10,18 +10,18 @@ namespace OrchardCore.Indexing
     /// </summary>
     public abstract class ContentPartIndexHandler<TPart> : IContentPartIndexHandler where TPart : ContentPart
     {
-        Task IContentPartIndexHandler.BuildIndexAsync(ContentPart contentPart, ContentTypePartDefinition typePartDefinition, BuildIndexContext context, ContentIndexSettings settings)
+        public async Task BuildIndexAsync(ContentPart contentPart, ContentTypePartDefinition typePartDefinition, BuildIndexContext context, ContentIndexSettings settings)
```

Comment:
Why are you adding `async`. It's better not to use the keyword when not necessary. Like in this case where the only statement to await is returned.

Reply:
I'll remove the async.